src/main/java/com/timgroup/statsd/StatsDProcessor.java

```diff
@@ -49,7 +48,15 @@
                 .onMalformedInput(CodingErrorAction.REPLACE)
                 .onUnmappableCharacter(CodingErrorAction.REPLACE);
 
-        public abstract void run();
+        public final void run() {
```

Comment:
Totally unrelated to this PR, but maybe we should add an exclusion for this type if we haven't done so already, just in case.